src/configs/hardware-config.c

```diff
@@ -259,7 +259,7 @@ static void init_hardware_config(void) {
 #endif
 
 const struct xnn_hardware_config* xnn_init_hardware_config() {
-  #if !XNN_PLATFORM_WEB && !XNN_ARCH_RISCV && !XNN_ARCH_PPC64 && XNN_ENABLE_CPUINFO
+  #if !XNN_PLATFORM_WEB && !XNN_ARCH_RISCV && !XNN_ARCH_PPC64 && !XNN_ARCH_HEXAGON && XNN_ENABLE_CPUINFO
```

Comment:
seems like the XNN_ENABLE_CPUINFOn should be off on platforms that do not want it?

Reply:
I think I prefer that as well. I needed to change in CMakeLists.txt:154

ADD_COMPILE_DEFINITIONS("XNN_ENABLE_CPUINFO=1")
to
ADD_COMPILE_DEFINITIONS("XNN_ENABLE_CPUINFO=$<BOOL:${XNNPACK_ENABLE_CPUINFO}>")

to be able to control this from build script. If you like other way better, please let me know!

Reply:
This changes failed other target build, so I changed this in CMakeLists.txt only setting XNN_ENABLE_CPUINFO=0 only for Hexagon, but others will be set to 1.